en/docs/design/rate-limiting/setting-throttling-limits.md

```diff
@@ -125,6 +125,18 @@ The default Rate Limiting levels are as follows:
 
 It is also possible to specify a bandwidth per unit time instead of a number of requests. This can be done through the Admin Portal of API Manager. For information on editing the values of the existing tiers, defining new tiers and specifying a bandwidth per unit time, see [Adding a new application-level rate limiting tier]({{base_path}}/design/rate-limiting/adding-new-throttling-policies/#adding-a-new-application-level-rate-limiting-tier).
 
+#### **Burst control**
```

Comment:
Is there a reason to bold this text after applying the header?

Reply:
Yeah, we are following the consistent pattern of document which is already available for subscription burst control. I think it was highligted as this is a key but an optional feature which customer should have idea of before implemented.